src/stirling/source_connectors/socket_tracer/redis_trace_bpf_test.cc

```diff
@@ -87,8 +87,8 @@ std::vector<RedisTraceRecord> GetRedisTraceRecords(
 TEST_F(RedisTraceBPFTest, VerifyBatchedCommands) {
   StartTransferDataThread();
 
-  constexpr std::string_view kRedisDockerCmdTmpl =
-      R"(docker run --rm --network=container:$0 redis bash -c "echo '$1' | redis-cli")";
+  constexpr std::string_view kRedisPodmanCmdTmpl =
+      R"(podman run --rm --network=container:$0 docker.io/redis bash -c "echo '$1' | redis-cli")";
```

Comment:
I take it the `docker` cli special cases the docker hub registry whereas podman doesn't?

Reply:
correct